backend/handler/database/roms_handler.py

```diff
@@ -510,7 +566,7 @@ def build_func(provider: str, column: InstrumentedAttribute):
             or selected_company
             or selected_age_rating
         ):
-            query = query.join(RomMetadata)
+            query = query.outerjoin(RomMetadata)
```

Comment:
@zurdi15 these `outerjoin` mean the Rom model is the base model used in the SQL query